### DISCUSSION TITLE
Custom network successfully added on trustwallet chrome browser, but on mobile apps its shown node connection error.

### DISCUSSION THREAD

**Discussion by mdd0x** [trustwallet/wallet-core#3134] [Q&A]
Dear Trustwallet team, 

im trying add custom network based on chainlist.org. its running successfully on trustwallet chrome browser. but its not working on mobile apps. 

here the screenshot
![image](https://user-images.githubusercontent.com/22166837/235529725-14c3a373-c608-4283-9a5c-c88c483a0745.png)

Thank you

**Comment by Milerius** [comment]
Hello, is it ios or android?

**Reply by mdd0x** [reply]
Hi, its for android @Milerius , some network its working fine. but its happend especially for besu node.
